### ISSUE TITLE
Crash reports

### ISSUE THREAD

**Issue by lalomartins** [raelgc/scudcloud#47]
In the last few days, I've been getting occasional “crash report” popups.

![screenshot](http://i.imgur.com/ktvTr53.png)

(In retrospect, I should have clicked “Show details”, sorry. Will do next time.)

However, ScudCloud is still running just fine. If I click “continue”, it will try to relaunch the app, which results in a dialogue telling me it's already running.

In my experience, these bogus crash reports happen because a child process segfaulted; no idea if that could be the case here, though. I'll attach the “details” next time it happens.

**Comment by raelgc** [comment]
Hi again, @lalomartins!

Yes, please, attach the details it and I can prevent this crash. I suspect is the same issue related to notify-osd restarting.

**Comment by lalomartins** [comment]
thought it might be, almost posted this as a comment on #26 but then I figured let's do it by the book :smile_cat:

**Comment by lalomartins** [comment]
I left the “send report” checkbox on but I have no idea if a report was sent and where :stuck_out_tongue_winking_eye:  But I think I captured the important info here:

![screenshot](http://i.imgur.com/v5MZsBr.png)

**Comment by raelgc** [comment]
Cool @lalomartins!

And this is exactly the issue related to Notify-OSD restart. Locally I tried a simple call to `notify_init` again, but it didn't work.

Anyway, good you got the exactly stack. Now I can focus in the root problem!

**Comment by lalomartins** [comment]
I thought nothing was broken, but now you mentioned I can confirm I'm getting no notifications, even though Slack didn't complain about permission

**Comment by raelgc** [comment]
I tried to restart the notifications with the following code in `/opt/scudcloud/lib/scudcloud.py` but the error appears to be deep then that:

``` python
    def notify(self, title, message, retry=True):
        try:
            notice = notify2.Notification(title, message, get_resource_path('scudcloud.png'))
            # Allow appending new message to existing notification.
            notice.set_hint_string('x-canonical-append', '')
            notice.show()
            self.alert()
        except dbus.exceptions.DBusException:
            if retry:
                notify2.uninit()
                notify2.init(self.APP_NAME)
                self.notify(title, message, False)
```

**Comment by raelgc** [comment]
Fixed in #93

**Comment by lalomartins** [comment]
fix tested and confirmed.
